shared/Microsoft.AspNetCore.Hosting.WebHostBuilderFactory.Sources/WebHostFactoryResolver.cs

```diff
@@ -0,0 +1,114 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+
+namespace Microsoft.AspNetCore.Hosting.WebHostBuilderFactory
+{
+    internal class WebHostFactoryResolver
+    {
+        public static readonly string CreateWebHostBuilder = nameof(CreateWebHostBuilder);
+        public static readonly string BuildWebHost = nameof(BuildWebHost);
+
+        public static FactoryResolutionResult ResolveWebHostBuilderFactory(Assembly assembly)
```

Comment:
I would expect EF wants wants a combine method they can call that supports both the 2.0 or 2.1 pattern.

Reply:
That is https://github.com/aspnet/Hosting/pull/1203/files#diff-572ec7cf75c906228766114371468f11R31